docs/quick-start.md

```diff
@@ -377,6 +379,7 @@ kind: VMAuth
 metadata:
   name: demo
 spec:
+  selectAllByDefault: true
```

Comment:
I suppose that `selectAllByDefault: true` and `userNamespaceSelector: {}, userSelector: {}` are doing the same thing, aren't it?

Reply:
yes, it could be skipped if selectAllByDefault is defined.